app/javascript/react/screens/App/Overview/components/Migrations/InProgressCard.js

```diff
@@ -4,7 +4,7 @@ import { Card, Grid, noop } from 'patternfly-react';
 
 const InProgressCard = ({ title, children, onClick, ...props }) => (
   <Grid.Col sm={12} md={6} lg={4}>
-    <Card matchHeight onClick={onClick} {...props}>
+    <Card id={`${title.props.children}-progress-card`} matchHeight onClick={onClick} {...props}>
```

Comment:
It seems that `title.props.children` is an array

<img width="851" alt="devtools_-_localhost_8080_migration" src="https://user-images.githubusercontent.com/15141412/43417531-3cc0a72e-9409-11e8-99cd-04baee39ed27.png">

Reply:
how'd yah manage that?

Reply:
ah, maybe in yours, the first array element is null, so you don't see what I'm seeing.  Check out the third console log

<img width="159" alt="devtools_-_localhost_8080_migration" src="https://user-images.githubusercontent.com/15141412/43418840-3997639a-940d-11e8-924c-25452aae12bc.png">

Reply:
Oh wait, but there is still a comma

<img width="798" alt="devtools_-_localhost_8080_migration" src="https://user-images.githubusercontent.com/15141412/43418920-72feffd0-940d-11e8-93c5-8880a4393a88.png">

Reply:
I see what the difference is now.  In your screenshot, your migrations are still in the pending state.  The ones on my screen are in the in progress state

<img width="1693" alt="manageiq__migration" src="https://user-images.githubusercontent.com/15141412/43419182-400628a0-940e-11e8-8d81-24bc809b3ebc.png">

Reply:
@michaelkro thanks for the db, able to seeee what you seee, n FIXED IT <3 🙇‍♀️